alyssabull/idk
# Refactor/code cleanup

## What does this PR do?
- Destructures props in all components
- Adds propTypes to all components with props
- Raises accessibility score 
- Adds full responsiveness
- Adds additional integration tests to raise test coverage from an average of 45% to 78%
- Adds sass styling

## Where should the reviewer start?
- All components with props to see destructuing and propTypes
- Dev tools to see accessibility score and responsiveness
- `App.test.js` to see additional integration tests
- `sass` folder for sass styles

## How should this be manually tested?
- Ensure functionality of app is still working after refactoring of props
- Run `lighthouse` audit in devTools to see accessibility score
- Run `npm test` to see 22 passing tests
- Run `npm test -- --coverage --watchAll=false` to see test coverage
